README.md

```diff
@@ -40,6 +40,7 @@ output "kibana_endpoint" {
 | cognito_identity_pool_id    | Cognito Identity Pool Id | `string` |  |
 | cognito_user_pool_id    | Cognito User Pool Id | `string` |  |
 | cognito_es_role    | IAM Role used by ElasticSearch to create Cognito Client and credentials | `string` | `service-role/CognitoAccessForAmazonES` |
+| cognito_auth_role    | IAM Role assumed by authenticated Cognito User | `string` |  |
```

Comment:
Should `cognito_es_role` above be removed?

Reply:
I don't think so, the user can still choose to use a different role for this
It is still used here:
```terraform
  cognito_options {
    enabled          = true
    identity_pool_id = "${var.region}:${var.cognito_identity_pool_id}"
    role_arn         = "arn:aws:iam::${var.account_id}:role/${var.cognito_es_role}"
    user_pool_id     = var.cognito_user_pool_id
  }
```